blocks/library/block/index.js

```diff
@@ -40,46 +41,53 @@ class ReusableBlockEdit extends Component {
 	}
 
 	startEditing() {
-		const { reusableBlock } = this.props;
+		const { reusableBlock, block } = this.props;
 
 		this.setState( {
 			isEditing: true,
 			title: reusableBlock.title,
+			attributes: block.attributes,
 		} );
 	}
 
 	stopEditing() {
 		this.setState( {
 			isEditing: false,
 			title: null,
+			attributes: null,
 		} );
 	}
 
 	setAttributes( attributes ) {
-		const { updateAttributes, block } = this.props;
-		updateAttributes( block.uid, attributes );
+		this.setState( ( prevState ) => {
+			if ( prevState.attributes !== null ) {
+				return { attributes: { ...prevState.attributes, ...attributes } };
+			}
+			return null;
```

Comment:
Is an explicit return value necessary here? Or could we drop this line altogether?

Reply:
👌 fixed in 5a36e93ce.